components/color-picker/__docs__/storybook-stories.jsx

```diff
@@ -117,6 +117,7 @@ storiesOf(COLOR_PICKER, module)
 			errorTextWorkingColor="Hex is invalid. Please correct this field."
 			id="working-color-error-state-color-picker"
 			labels={{ label: 'Choose Color' }}
+			valueWorking="#f"
```

Comment:
Story showing error message on working color should use an invalid color for better representation.